pkg/network/netbinding/netbinding_test.go

```diff
@@ -108,10 +108,19 @@ var _ = Describe("Network Binding", func() {
 		})
 	})
 
-	Context("network binding plugin configuration", func() {
+	Context("slirp binding", func() {
```

Comment:
Please add a test for a VM with slirp interface and some plugin interface,

Reply:
Added with a commit at the tail.
Done